core/lib/libfdt/fdt_rw.c

```diff
@@ -285,7 +285,8 @@ int fdt_setprop(void *fdt, int nodeoffset, const char *name,
 	if (err)
 		return err;
 
-	memcpy(prop->data, val, len);
+	if (len)
+		memcpy(prop->data, val, len);
```

Comment:
The compiler (gcc 5.3) issues some runtime checks with `-fsanitize=undefined` which is triggered here if `val` is NULL.  I'd rather there was some other way around it.